mParticle-Apple-SDK/Logger/MPLogger.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MPLogger.swift
+//  mParticle-Apple-SDK
+//
+//  Created by Brandon Stalnaker on 9/22/24.
+//
+
+import Foundation
+
+public struct MPLogger {
+    
+    public static func MPLogger(loggerLevel: MPILogLevel, format: String, arguments: any CVarArg...) {
+        if (MParticle.sharedInstance().logLevel.rawValue >= loggerLevel.rawValue && loggerLevel != .none) {
+            let msg = String.localizedStringWithFormat(format, arguments)
+// Custom Logger is marked as nonnull despite the fact that we expect it to be null if not set by the client
+//         if let customLogger = MParticle.sharedInstance().customLogger {
+//             customLogger(msg)
+//         } else
```

Comment:
```suggestion
```
I assume this was meant to be deleted?

Reply:
no I wanted to leave this here to make sure we understood this is kinda a todo once we fully convert to Swift